vid/src/avid_m/namespaced.rs

```diff
@@ -169,6 +171,12 @@ impl NsAvidMScheme {
         if shares.is_empty() {
             return Err(VidError::InsufficientShares);
         }
+        if shares
+            .iter()
+            .any(|share| ns_id >= share.ns_lens.len() || ns_id >= share.content.len())
```

Comment:
`ns_id >= share.content.len()`, why this?

Reply:
avoid panic in later `share.inner_ns_share(ns_id)` and `shares[0].ns_commits[ns_id]`

Reply:
got it, can you add a comment about how the content is expected to be as long as the number of namespace, even if the namespace has empty payload, they should have a non-empty `RawAvidShare` with some metadata?